openml-datarobot/src/test/java/com/feedzai/openml/datarobot/ClassificationBinaryDataRobotModelTest.java

```diff
@@ -0,0 +1,48 @@
+package com.feedzai.openml.datarobot;
+
+import com.feedzai.openml.data.schema.DatasetSchema;
+import com.feedzai.openml.mocks.MockInstance;
+import com.feedzai.openml.provider.exception.ModelLoadingException;
+import com.google.common.primitives.Doubles;
+import org.junit.Test;
+
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests for classifying DataRobot Model with {@link ClassificationBinaryDataRobotModel}.
+ *
+ * @author Sheng Wang (sheng.wang@feedzai.com)
+ * @since 1.0.6
+ */
+public class ClassificationBinaryDataRobotModelTest extends DataRobotModelProviderLoadTest {
+
+    /**
+     * Checks the method ClassificationBinaryDataRobotModel#classify() if it ensures
+     * that DataRobot classifies correctly the index of maximum value of the scores' list.
+     *
+     * @throws ModelLoadingException If the model cannot be loaded.
+     *
+     * @since 1.0.6
```

Comment:
We do not need since annotations everywhere unless they are newer than the one of the class. It is the common standard.